modules/installation-configuration-parameters.adoc

```diff
@@ -1591,8 +1603,76 @@ Additional VMware vSphere configuration parameters are described in the followin
 |====
 |Parameter|Description|Values
 
+|`platform.vsphere`
+| Parameter covers two purposes in the context of a vSphere configuration: (1) Describes your account on the cloud platform that hosts your cluster, and you use the parameter to customize the platform; (2) The parameter is optional when providing additional configuration settings for the machine pool parameters associated with compute and control plane machines.
+a|String
+[NOTE]
+====
+You can only specify one vCenter server for your {product-title} cluster.
+====
+
+|`platform.vsphere.vCenters`
+|Lists any fully-qualified hostname or IP address of a vCenter server.
+|String
+
+|`platform.vsphere.datacenters`
```

Comment:
platform.vsphere.datacenters also not a valid field, seem like it should be platform.vsphere.vcenters.datacenters, but should we also list other fields under vcenters here?
```
sh-4.4$  ./openshift-install explain installconfig.platform.vsphere.datacenters
FATAL Error executing openshift-install: failed to load schema for the field platform.vsphere.datacenters: invalid field datacenters, no such property found
```

Reply:
Updated.  @rvanderp3 , WDYT of the comment `but should we also list other fields under vcenters here?` ?

Reply:
yep, good catch @sgaoshang , we should list datacenters as well under vcenters

Reply:
Sorry, @sgaoshang and @rvanderp3 . What do I need to list in the table regarding `installconfig.platform.vsphere.datacenter`?